Readme.md

```diff
@@ -3,19 +3,27 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* **[PopupListViewController.cs](./CS/EFCore/PopupActionEF/PopupActionEF.Module/Controllers/PopupListViewController.cs)**
-<!-- default file list end -->
-# How to add custom buttons ( Actions ) to the lookup and popup windows?
+# XAF - How to add custom buttons ( Actions ) to the lookup and popup windows
 
+You can show custom buttons ([Actions](https://docs.devexpress.com/eXpressAppFramework/112622/ui-construction/controllers-and-actions/actions)) near the **New**, **Clear**, **OK**, and **Cancel** buttons in lookup and popup List Views. For this purpose, create a new View Controller with a [Simple Action](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Actions.SimpleAction) and set its [`Category`](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Actions.ActionBase.Category) property to `PopupActions`.
 
-<p>To add custom buttons near the New, Clear (OK, Cancel) buttons in the lookup/popup list views, it is enough to add a new view controller with two simple actions, and set its Category property to the <strong>PopupActions</strong> value.<br />
-This example demonstrates how to add an action to open a selected object directly from a popup/lookup list view.</p><p>Refer to the <a href="http://documentation.devexpress.com/#Xaf/CustomDocument2804"><u>Add Actions to a Pop-up Window</u></a> help topic for more information.</p>
+This example adds a new **Action In Popup** Action to popup/lookup List Views. When a user clicks this Action, the application displays a sample message box.
 
-<br/>
+Refer to the following topic for more information: [Add Actions to a Pop-up Window](https://docs.devexpress.com/eXpressAppFramework/112804/ui-construction/controllers-and-actions/add-actions-to-a-popup-window).
 
 ![image](https://user-images.githubusercontent.com/14300209/225921128-c14fc411-d304-4cb8-9d47-2080f2d1ec86.png)
-![image](https://user-images.githubusercontent.com/14300209/225921293-f5c3f810-90e4-4697-bcd6-d711406a6e00.png)
 
+## Files to Review
+
+* [PopupListViewController.cs](./CS/EFCore/PopupActionEF/PopupActionEF.Module/Controllers/PopupListViewController.cs)
+
+## Documentation
+
+* [How to: Reorder an Action Container's Actions Collection](https://docs.devexpress.com/eXpressAppFramework/112815/ui-construction/controllers-and-actions/actions/how-to-reorder-an-action-containers-actions-collection)
+* [How to: Place an Action in a Different Location](https://docs.devexpress.com/eXpressAppFramework/402145/ui-construction/controllers-and-actions/actions/how-to-place-an-action-in-a-different-location)
+* [Action Containers](https://docs.devexpress.com/eXpressAppFramework/112610/ui-construction/action-containers)
```

Comment:
```suggestion
* [Action Containers](https://docs.devexpress.com/eXpressAppFramework/112610/ui-construction/action-containers)
* [Actions](https://docs.devexpress.com/eXpressAppFramework/112622/ui-construction/controllers-and-actions/actions)
```

Reply:
I'd add a link to the actions topic since we have linked it in our text. It's up to you